app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/agenda_screen/AgendaState.kt

```diff
@@ -17,9 +17,17 @@ data class AgendaState(
 
     val errorMessage: UiText? = null,
 
-    // Dropdown Menus
+    // Menu for logout
     val isLogoutDropdownVisible: Boolean = false,
+
+    // Menu for editing/viewing/deleting agenda items
     val agendaItemIdForMenu: UuidStr? = null,
 
+    // Menu for adding agenda items
+    val isAddAgendaItemDropdownVisible: Boolean = false,
+
+    // One-time events
+    val oneTimeEvent: AgendaEvent? = null,
```

Comment:
I also find the name `oneTimeEvent` misleading, since it's modeled as state which is everything but one time

Reply:
I have it as a stateful one-time event. I have a reset, and it similar to your pattern of a `ShowXXXXEvent` and `SeenXXXEvent`, just more generalized.

Reply:
Wrote a bit about that in my first comment, but how do you know there can only be one one-time event at a time? Couldn't it be that a snackbar shows while you perform a scroll to the latest item?